test/ssl/test.sh

```diff
@@ -141,7 +143,7 @@ die() {
 }
 
 admin() {
-	psql -X -h /tmp -U pgbouncer pgbouncer -c "$@;" || die "Cannot contact bouncer!"
+	psql -X -h /tmp -U pgbouncer -d pgbouncer -c "$@;" || die "Cannot contact bouncer!"
```

Comment:
Without this change the tests would now show errors similar to the ones reported here: https://www.postgresql.org/message-id/flat/a1a18525-6b24-411d-860c-404878c73e97%40k4g2000yqb.googlegroups.com#a65e9ecbf62f6a12fd9498daef0d2414

Reply:
I guess this started happening because of this commit: https://github.com/pgbouncer/pgbouncer/commit/c7eb9fede68bf2131ef4442ca923f4498a453b04

Reply:
Fixed separately: d617fbe7148f7b8a4d5decc9b2866af845879250